README.md

```diff
@@ -122,6 +122,8 @@ Enable the rules that you would like to use.
 * [react/no-find-dom-node](docs/rules/no-find-dom-node.md): Prevent usage of findDOMNode
 * [react/no-is-mounted](docs/rules/no-is-mounted.md): Prevent usage of isMounted
 * [react/no-multi-comp](docs/rules/no-multi-comp.md): Prevent multiple component definition per file
+* [react/no-named-import](docs/rules/no-named-import.md): No named imports on React
+* [react/no-namespace-import](docs/rules/no-namespace-import.md): No namespace import on React (fixable)
```

Comment:
this is already a generic rule, and should not just apply to react: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-namespace.md

Reply:
Ah ofcourse, I'll remove it.